src/server/database/config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
-  databaseName : 'layouts',
+  databaseName : 'halfBaked',
```

Comment:
@hmtinc @gaelder the current config for the db is:
```js
  databaseName : 'metadataTestMk2',
  ip: '127.0.0.1',
```

Which one should be kept?

Reply:
Oops, my changes there should have been reverted. It should be 
  databaseName : 'metadataTestMk2',
  ip: '127.0.0.1',